lib/matplotlib/artist.py

```diff
@@ -1580,5 +1582,5 @@ def kwdoc(a):
 
 docstring.interpd.update(Artist=kwdoc(Artist))
 
-_get_axes_msg = """This has been deprecated in mpl 1.5, please use the
+_get_axes_msg = """{} has been deprecated in mpl 1.5, please use the
```

Comment:
Right, so the point is that `'{}'` can't be used in v1.5.x branch. Same thing with my comment about the backport I just merged into 1.5.x.